src/api/app/assets/stylesheets/webui2/icons.scss

```diff
@@ -0,0 +1,7 @@
+.icons-sponsor_amd { width: 90px; height: 50px; background: image-url('icons_sprite.png') -126px -177px no-repeat; }
```

Comment:
Why using the sprite instead of individual images? Definitely out of scope but I really would like to turn this into a model with a configuration interface so I don't have to be root to be able to add/remove/change a sponsor...

Reply:
I would like to do it too, and to use a carousel to show instead of one random one on each load of the page... but I'm just trying to fix what you did in your PR. I think we can start changing many things in other PRs to keep this as simple as possible.

Reply:
Sure I understand. But why do you use the sprite if you have the images in `app/assets/icons`?

Reply:
I didn't use it... it was there already, I just copied the needed CSS classes. You used the sprites as before :dagger: 
But anyway,... if I have time I will change that to use the images directly

Reply:
Yeah I missed that partial in my previous PR somehow. It's really hard to get all the conditions...

Reply:
Let's not add more things in this PR and create a card for the new way of adding sponsors... do you agree?

Reply:
Of course.